cmd/gosky/main.go

```diff
@@ -823,6 +824,29 @@ var resetPasswordCmd = &cli.Command{
 	},
 }
 
+var updateHandleCmd = &cli.Command{
```

Comment:
It would be nice to pull out commits like this into separate PRs. And the `--auth` revert to a seperate commit.

Reply:
Yeah, i'm moving towards that, but things are still in the phase of needing a bunch of things to land all at the same time to stay up to date.